jsons/parameters.json

```diff
@@ -15897,5 +16149,840 @@
+  },
+  {
+    "_id": 392,
+    "default_value": "",
+    "description": "This parameter controls whether the output text file, as controlled by the parameter output_txtfile, will have a default \".txt\" file extension or a custom extension specified by this parameter.",
+    "key_translations": {
+      "comet_style_1": "text_file_extension",
+      "ursgal_style_1": "text_file_extension"
+    },
+    "name": "text_file_extension",
+    "tag": [
+      "output"
+    ],
+    "triggers_rerun": false,
+    "value_translations": {},
+    "value_type": "str"
+  },
+  {
+    "_id": 393,
+    "default_value": 0,
+    "description": "This parameter controls how the peptide_mass_tolerance parameter is applied. That tolerance can be applied to the singly charged peptide mass (default=0) or it can be applied to the precursor m/z (1).",
+    "key_translations": {
+      "comet_style_1": "precursor_tolerance_type",
+      "ursgal_style_1": "precursor_tolerance_type"
+    },
+    "name": "precursor_tolerance_type",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 394,
+    "default_value": 0,
+    "description": "This parameter specifies how theoretical fragment ion peaks are represented.",
+    "key_translations": {
+      "comet_style_1": "theoretical_fragment_ions",
+      "ursgal_style_1": "theoretical_fragment_ions"
+    },
+    "name": "theoretical_fragment_ions",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 395,
+    "default_value": [
+      600.0,
+      8000.0
+    ],
+    "description": "Defines the mass range of peptides to search (based on MH+ or the singly protonated mass).",
+    "key_translations": {
+      "comet_style_1": "digest_mass_range",
+      "ursgal_style_1": "digest_mass_range"
+    },
+    "name": "digest_mass_range",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "list"
+  },
+  {
+    "_id": 396,
+    "default_value": 20,
+    "description": "This parameter defines the maximum number of proteins (identifiers/accessions) to report. If set to -1, there will be no limit on the number of reported additional proteins.",
+    "key_translations": {
+      "comet_style_1": "max_duplicate_proteins",
+      "ursgal_style_1": "max_duplicate_proteins"
+    },
+    "name": "max_duplicate_proteins",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 397,
+    "default_value": 0,
+    "description": "This parameter sets the maximum indexed database search run time for a scan/query.",
+    "key_translations": {
+      "comet_style_1": "max_index_runtime",
+      "ursgal_style_1": "max_index_runtime"
+    },
+    "name": "max_index_runtime",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 398,
```

Comment:
Not sure if this really needs to be included, since it's an internal param. If yes, I'd suggest a name like "comet_internal_num_results"

Reply:
I changed it to "comet_internal_num_results". I agree that this is an internal param and we most likely are not going to make use of it in ursgal. However, based on the documentation it might be handy to set it for further processing with PeptideProphet (and Percolator, though the comment in the doc at this step is "not likely"). Hence, if someone wants to run comet via ursgal2 and then afterwards use PeptideProphet, this parameter is apparently helpful(?).